view/adminhtml/templates/element/actions.phtml

```diff
@@ -0,0 +1,56 @@
+<?php
+/** @var \Magento\Framework\Escaper $escaper */
+/** @var \Hyva\Admin\Block\Grid\Actions $block */
```

Comment:
Please adjust the type hint to be a regular `\Magento\Framework\View\Element\Template`.